### PR TITLE
Svelte: don't add `svelte-loader` nor `svelte` when initializing Svelte projects

### PR DESCRIPTION
**Pull request by benmccann**
## What I did

 `svelte-loader` should not be added indiscriminately. It is only needed for webpack projects, but Svelte projects use Vite by default, so it's mostly wrong to add it. Users will have set this up already if they're using Svelte and webpack

## How to test

- [ ] Is this testable with Jest or Chromatic screenshots?
- [ ] Does this need a new example in the kitchen sink apps?
- [ ] Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by MichaelArestad** [comment]
@j3rem1e can you please review this when you get a chance?

**Review by j3rem1e — APPROVED** [review]

**Comment by JReinhold** [comment]
Isn't this superseded by https://github.com/storybookjs/storybook/pull/19263 by @IanVS, that only adds the `svelte-loader` in Webpack based projects?

Or do we not want to add the loader at all, because we assume webpack setups already have that? If they already have that then this won't make a difference though, right?
If we never want to add it - not even for webpack projects - wouldn't the same hold true for the `svelte` package, don't we expect that to be present too always?


I haven't tried any of this out, but my hunch tells me that we shouldn't add any extra packages at all, because we can assume that they're already there, since they are actually essential packages and not extra packages.

File: code/lib/cli/src/generators/SVELTE/index.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { baseGenerator } from '../baseGenerator';
 import type { Generator } from '../types';
 
 const generator: Generator = async (packageManager, npmOptions, options) => {
-  const extraPackages = options.builder === CoreBuilder.Webpack5 ? ['svelte', 'svelte-loader'] : [];
+  const extraPackages = options.builder === CoreBuilder.Webpack5 ? ['svelte'] : [];
 
   await baseGenerator(packageManager, npmOptions, options, 'svelte', {
     extraPackages,
```
Comment (benmccann) [inline code/lib/cli/src/generators/SVELTE/index.ts:9]:
```suggestion
```

File: code/lib/cli/src/generators/SVELTE/index.ts [hunk 2]
```diff
@@ -3,10 +3,9 @@ import { baseGenerator } from '../baseGenerator';
 import type { Generator } from '../types';
 
 const generator: Generator = async (packageManager, npmOptions, options) => {
-  const extraPackages = options.builder === CoreBuilder.Webpack5 ? ['svelte', 'svelte-loader'] : [];
+  const extraPackages = options.builder === CoreBuilder.Webpack5 ? ['svelte'] : [];
```
Comment (benmccann) [inline code/lib/cli/src/generators/SVELTE/index.ts:6]:
```suggestion
```

**Comment by benmccann** [comment]
I would assume both svelte-loader and Svelte are already present and agree we shouldn't need to add either of them. I've updated this PR accordingly

**Review by JReinhold — APPROVED** [review]

**Pull request merged**